oar/core/operators.py

```diff
@@ -140,6 +140,47 @@ def approve_release(self) -> None:
             raise
 
 
+class ImageHealthOperator:
+    """Handles image health check operations for both advisory and shipment data"""
+    
+    def __init__(self, cs: ConfigStore):
+        self._am = AdvisoryManager(cs)
+        self._sd = ShipmentData(cs)
+        self._nm = NotificationManager(cs)
+
+    def check_image_health(self) -> bool:
+        """
+        Check image container health (grade info) handling different release flows:
+        - Konflux: Checks shipment data and adds comment to MR
+        - Errata: Checks advisory data and sends notifications for unhealthy containers
+        
+        Returns:
+            bool: True if all image containers are healthy (grade A), False otherwise
```

Comment:
```suggestion
            bool: True if all image containers are healthy (grade A or B), False otherwise
```
It is still A or B, isn't it